### PR TITLE
update infer  preprocessing

### PR DESCRIPTION
**Pull request by magicwifi**
There are some problems with the preprocessing on LPRnet when infer, which will cause the predicted output results to be inconsistent with the pytorch version; as shown, the preprocessing has been updated here

### CODE REVIEW

**Comment by wang-xinyu** [comment]
Is it related to issue https://github.com/wang-xinyu/tensorrtx/issues/622?

**Comment by wang-xinyu** [comment]
@chgit0214  could you help review?

**Comment by magicwifi** [comment]
> Is it related to issue #622?

yes, my friend find this issue, and i fix it
> Is it related to issue #622?

**Comment by wang-xinyu** [comment]
@magicwifi thanks, let's wait @chgit0214 reply.

File: lprnet/LPRnet.cpp [hunk 1]
```diff
@@ -399,8 +399,19 @@ int main(int argc, char **argv) {
...
+    for (int row = 0; row < INPUT_H; ++row) {
+        uchar* uc_pixel = pr_img.data + row * pr_img.step;
+        for (int col = 0; col < INPUT_W; ++col) {
+            data[i + 2 * INPUT_H * INPUT_W] = ((float)uc_pixel[2] - 127.5)*0.0078125;
+            data[i + INPUT_H * INPUT_W] = ((float)uc_pixel[1]-127.5)*0.0078125;
+            data[i] = ((float)uc_pixel[0]-127.5)*0.0078125;
+            uc_pixel += 3;
+            ++i;
+        }
+    }
```
Comment (wang-xinyu) [inline lprnet/LPRnet.cpp:414]: Do you mean the input should be B G R not R G B?

Comment (magicwifi) [reply lprnet/LPRnet.cpp:414]: the point is [https://github.com/xuexingyu24/License_Plate_Detection_Pytorch/blob/master/LPRNet/LPRNet_Test.py](url)  88 line np.float then transpose the normalized from -1 to 1  is differenet from the code in [https://github.com/wang-xinyu/tensorrtx/blob/master/lprnet/LPRnet.cpp](url) in 402 line blobFromImage; update my process, the value  input  tensorrt lpr model is equal to  [https://github.com/xuexingyu24/License_Plate_Detection_Pytorch/blob/master/LPRNet/LPRNet_Test.py](url)  input the lpr pytorch model;  please check

**Review by wang-xinyu — APPROVED** [review]

**Pull request merged**